src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -5410,6 +5421,59 @@ public String toString()
             return "GuildBan:" + user + (reason == null ? "" : '(' + reason + ')');
         }
     }
+    /**
+     * Represents the NSFW level for this guild.
+     * <p>
+     * <br><b>DEFAULT</b>   {@literal ->}  Is yet to be classified by discord.
+     * <br><b>EXPLICIT</b>  {@literal ->} Is classified as a NSFW server
+     * <br><b>SAFE</b>  {@literal ->}  Doesn't classify as a NSFW server
+     * <br><b>AGE_RESTRICTED</b>  {@literal ->} Is classified as NSFW and has an age restriction in place
```

Comment:
Move these descriptions to the enum constants instead

Reply:
MFALevel has the descripctions like that. Should I also move them to the constants?